db.php

```diff
@@ -289,8 +289,7 @@ function chooseMentor() {
 
 
 	 function getMentor($mentor) {
-		global $_USER;
-		$user = $_USER['uid'];
+		$user = $mentor;
```

Comment:
Previously "getMentor" was actually not returning the user's mentor, it was just returning the logged in user. This change does mean that you can use getMentor to get the information for any mentor, but it's actually necessary to get the data for wishlist/search previews too. If we want to change it we'll have to make a second api call. One to get the full information for a currently matched mentor, one to get partial information for any mentor.